### DISCUSSION TITLE
Captive Portal

### DISCUSSION THREAD

**Discussion by potros13** [khoih-prog/ESPAsync_WiFiManager_Lite#7] [Ideas]
Hi!

I see that Captive Portal feature has been removed in this "Lite" version. Any specific reason for this? Any posibility to get it included? I believe this is very confortable feature to lauch the credentials page directly without the need to know the Server IP.

Thanks

**Comment by khoih-prog** [comment]
Hi @potros13 

Thanks for using the library and your feature request.

As you know this library only shares the **ESPAsync_WiFiManager** library's name, but is written to support simpler and *Lite** applications, without the sophisticated Json.

That's why no Captive-Portal feature was included intentionally.

Certainly, if this feature is requested by more users, I'll add later as I really don't like to spend time to work on a not popular feature.

Let's wait and see.

**Comment by hmueller01** [comment]
Implemented by #22.
